oscillatingCollimatorApp/Db/oscillatingCollimator_Let.db

```diff
@@ -92,3 +92,20 @@ record(calc, "$(P)$(O):ANGLE") {
     field(ASG, "READONLY")
 }
 
+record(bo, "$(P)$(O):USES_LSR")
+{
+	field(DESC, "Whether the inst. uses the laser")
+	field(VAL, "1")
+}
+record(bi, "$(P)$(O):COLLIM_MOVING")
+{
+	field(DESC, "Whether the collimator is moving")
+	field(INP, "$(P)STAB:IS_STABLE CP MS")
```

Comment:
I think we could probably move 
https://github.com/ISISComputingGroup/EPICS-motorExtensions/blob/master/oscillatingCollimatorApp/Db/oscillatingCollimator.db#L175-L222
out into the wish specifi one, and use your logic for the LET + MERLIN ones, but using the same (`MOVING`) pv name.  I think that also means the OPI can be simplified to always show whether or not the collimator is moving or not. 

PS - I might be being blind but can't see where the STAB PVs are stored, are they in a separate db file?

Reply:
i am being blind, theyre in utilities... 🤦‍♂️